spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/GatewayApiController.java

```diff
@@ -70,7 +71,7 @@ public ServiceApi findServiceByApiKeyValue(@RequestParam("apiKeyValue") String a
 
     //Due to legacy code reasons the gateway API key and the gateway id are exactly the same.
     @GetMapping("/api/gateways/api-keys")
-    public ResponseEntity<List<ApiKey>> findApiKeyListByGatewayApiKey(@RequestParam("gateway-api-key") String gatewayIdAndApiKey) {
+    public ResponseEntity<List<HashedApiKey>> findApiKeyListByGatewayApiKey(@RequestParam("gateway-api-key") String gatewayIdAndApiKey) {
```

Comment:
This breaks the existing API, so we should discuss if we should really change this endpoint to return hashed API keys, or if we want to add a new endpoint, while this one stays for backwards compatibility.

Reply:
As discussed, backwards compatibility is not required.